pkg/controller/directimagemigration/task.go

```diff
@@ -202,6 +201,7 @@ func (t *Task) Run() error {
 			if len(failedISList) > 0 {
 				t.fail(MigrationFailed, failedISList)
 			} else {
+				t.Requeue = PollReQ
```

Comment:
PollReQ works here. You can also just not set it and it will stay as the default `FastReQ` I believe.

Reply:
Actually, not sure what the default is in the DIM controller task.go